card/src/reducer/card.reducer.js

```diff
@@ -0,0 +1,27 @@
+const cardReducer = (
+  state = {
+    loading: false,
+    userData: null,
+    error: null,
+    status: null
+  },
+  action
+) => {
+  switch (action.type) {
+    case "CARD_REQUESTING":
```

Comment:
export this from action 
import * as cardAction from "../action/card.action.js
ex:-cardAction.CARD_REQUESTING

Reply:
done